package io.gitub.msavaliadorcredito.msavaliadorcredito.application.exception;

public class DadosClienteNotFoundException extends  Exception {
    public DadosClienteNotFoundException() {
        super("Dados Cliente n√£o encontrado para o cpf informado.");
    }
}
